### ISSUE TITLE
Imputing missing phenotypes

### ISSUE THREAD

**Issue by zmy24** [francois-a/fastqtl#19]
Hi, I have a phenotype data with some missing value. And I found the Fastqtl will impute the issing phenotypes automatically. I wonder if it was imputed by the average value? And how can I  avoid imputation?

**Comment by francois-a** [comment]
Yes, missing phenotypes are [mean-imputed](https://github.com/francois-a/fastqtl/blob/47180b9a0ef4e806b9fe7e7830d6f4c1181567a0/src/management.cpp#L254). To avoid this, you either need to filter out phenotypes with missing values, or impute them using an alternate method prior to running fastQTL.
